lib/spark_api.rb

```diff
@@ -1,6 +1,7 @@
 require 'rubygems'
 require 'logger'
 require 'multi_json'
+require 'addressable'
```

Comment:
This works fine in command line.

Reply:
Looks good!

Reply:
Do I need to bump up the gem version just for this?